lib/parser.js

```diff
@@ -190,6 +195,7 @@ async function validateAndConvertMessage(msg, originalAsyncAPIDocument, fileForm
   });
 
   msg.schemaFormat = DEFAULT_SCHEMA_FORMAT;
+  msg[String(xParserOriginalSchemaFormat)] = schemaFormat;
```

Comment:
code from https://github.com/asyncapi/parser-js/pull/214/files#diff-2f45270c7389933f71b380538ff52e106ee5cb66a0b573d9967467dfdbc11c02R187 runs a message parser for a given mime type, so if in your AsyncAPI file you have a message that has schema provided in avro, then this code runs -> https://github.com/asyncapi/parser-js/pull/214/files#diff-2f45270c7389933f71b380538ff52e106ee5cb66a0b573d9967467dfdbc11c02R187

- 1st the avro parser puts avro mime type under `x-parser-original-schema-format` 
- 2nd you override it with default schema format, asyncapi one, so we loose information about the original schema format

This is why instead of putting this line here, you need to have it in this parser of default asyncapi schema -> https://github.com/asyncapi/parser-js/blob/master/lib/asyncapiSchemaFormatParser.js

Add support for https://github.com/asyncapi/avro-schema-parser/blob/master/index.js#L7 too and put info about it in the readme as well please 🙏🏼